ethportal-peertest/src/utils.rs

```diff
@@ -132,29 +156,100 @@ pub fn fixture_receipts() -> (HistoryContentKey, HistoryContentValue) {
+    read_binary_history_fixture(15040641, None)
 }
 
-fn read_epoch_acc(hash: &str) -> (HistoryContentKey, HistoryContentValue) {
-    let epoch_acc = std::fs::read(format!("test_assets/mainnet/0x03{hash}.portalcontent")).unwrap();
-    let epoch_acc_hash = ethportal_api::utils::bytes::hex_decode(&format!("0x{hash}")).unwrap();
-    let content_key =
-        ethportal_api::HistoryContentKey::EpochAccumulator(ethportal_api::EpochAccumulatorKey {
-            epoch_hash: alloy_primitives::B256::from_slice(&epoch_acc_hash),
-        });
-    let content_value = HistoryContentValue::decode(&content_key, &epoch_acc).unwrap();
-    (content_key, content_value)
+/// History BlockBody content key & value
+/// Block #15040641 (pre-merge)
+pub fn fixture_block_body_15040641() -> (HistoryContentKey, HistoryContentValue) {
+    read_binary_history_fixture(15040641, Some(DependentType::BlockBody))
+}
+
+/// History Receipts content key & value
+/// Block #15040641 (pre-merge)
+pub fn fixture_receipts_15040641() -> (HistoryContentKey, HistoryContentValue) {
+    read_binary_history_fixture(15040641, Some(DependentType::Receipts))
+}
+
+/// History HeaderWithProof content key & value
+/// Block #15040708 (pre-merge)
+pub fn fixture_header_by_hash_with_proof_15040708() -> (HistoryContentKey, HistoryContentValue) {
+    read_binary_history_fixture(15040708, None)
+}
+
+/// History BlockBody content key & value
+/// Block #15040708 (pre-merge)
+pub fn fixture_block_body_15040708() -> (HistoryContentKey, HistoryContentValue) {
+    read_binary_history_fixture(15040708, Some(DependentType::BlockBody))
+}
+
+/// History Receipts content key & value
+/// Block #15040708 (pre-merge)
+pub fn fixture_receipts_15040708() -> (HistoryContentKey, HistoryContentValue) {
+    read_binary_history_fixture(15040708, Some(DependentType::Receipts))
+}
+
+fn read_binary_history_fixture(
+    block_number: u64,
+    dependent: Option<DependentType>,
+) -> (HistoryContentKey, HistoryContentValue) {
+    let header_value = std::fs::read(format!(
+        "test_assets/mainnet/large_content/{block_number}/header.bin"
+    ))
+    .unwrap();
+    let header_content_value: HeaderWithProof =
+        HeaderWithProof::from_ssz_bytes(&header_value).unwrap();
+
+    match dependent {
+        Some(dependent_type) => {
+            let dependent_value = std::fs::read(format!(
+                "test_assets/mainnet/large_content/{block_number}/{dependent_type}.bin"
+            ))
+            .unwrap();
+            match dependent_type {
+                DependentType::BlockBody => {
+                    let content_key = HistoryContentKey::BlockBody(BlockBodyKey {
+                        block_hash: header_content_value.header.hash().0,
+                    });
+                    let content_value =
+                        HistoryContentValue::decode(&content_key, &dependent_value).unwrap();
```

Comment:
shouldn't clippy be complaining about this unwrap?

Reply:
This is in `ethportal-peertest`, I thought testing stuff didn't have that check